lib/fastlane/plugin/wpmreleasetoolkit/actions/common/upload_to_s3.rb

```diff
@@ -0,0 +1,107 @@
+require 'fastlane/action'
+require 'digest/sha1'
+
+module Fastlane
+  module Actions
+    module SharedValues
+      UPLOADED_FILE_PATH = :UPLOADED_FILE_PATH
```

Comment:
Given `SharedValues` is a wide and quite global namespace I'd  suggest use a more specific constant name here.

```suggestion
      S3_UPLOADED_FILE_PATH = :S3_UPLOADED_FILE_PATH
```

Reply:
Addressed in 1dcc36bb4c5cb9d682dbcd6b762b3d29f632e1be to also update its usage.